website/docs/index.html.markdown

```diff
@@ -293,6 +321,13 @@ The following arguments are used to configure the VMware Cloud Director Provider
    environment variable. This token requires at least VCD 10.3.1. There are restrictions to its use, as defined in
    [the documentation](https://docs.vmware.com/en/VMware-Cloud-Director/10.3/VMware-Cloud-Director-Service-Provider-Admin-Portal-Guide/GUID-A1B3B2FA-7B2C-4EE1-9D1B-188BE703EEDE.html)
 
+* `service_account_token_file` - (Optional; *v3.9+*) This is the file that contains a Service Account API token. The
```

Comment:
Question, do we still try to pull this into v3.9.0 or just move it to v3.10.0? Will keep this unresolved until we reach a decision when it is good to get merged.

Reply:
Got it, introduced a changelog entry.